telegram/ext/filters.py

```diff
@@ -2545,3 +2547,51 @@ def filter(self, message: Message) -> bool:
 
 VOICE = _Voice("filters.VOICE")
 """Messages that contain :attr:`telegram.Message.voice`."""
+
+
+class Mention(MessageFilter):
+    """Mention messages. If a username is passed, it filters messages to only allow those
+    whose contains mention of username.
+
+    Examples:
+        A simple use case for passing username is to allow only messages that were sent with
+        mention username:
+        MessageHandler(filters.MENTION("username"), callback_method)
+        MessageHandler(filters.MENTION, callback_method)
+
+    Args:
+        username (:obj:`str`, optional): Which messages to allow with username. Only
+            exact matches are allowed. If not specified, will allow any text message with mention.
+    """
+
+    __slots__ = ("username",)
+    entity_type: str = MessageEntity.MENTION
+
+    def __init__(self, username: Optional[Union[List[str], Tuple[str, ...]]] = None):
+        self.username: Optional[Sequence[str]] = username
+        super().__init__(
+            name=f"filters.Regex(r'@{username}') & filters.Entity({self.entity_type})"
```

Comment:
Please use somethling like `f"filters.Mention({username})"`, as `Regex(…) & Entity(…)` is not what this filter is doing